apple/internal/linking_support.bzl

```diff
@@ -157,12 +157,20 @@ def _register_linking_action(
         linkopts.extend(["-bundle_loader", bundle_loader_file.path])
         link_inputs.append(bundle_loader_file)
 
+    # TODO: This is a hack to support bazel 5.x and 6.x at the same time after
```

Comment:
I assume we want to create an issue to have this removed once we remove support for 5.x?

Reply:
I'll submit a revert as soon as this is in

Reply:
https://github.com/bazelbuild/rules_apple/pull/1386